ddtrace/bootstrap/sitecustomize.py

```diff
@@ -102,7 +102,9 @@ def add_global_tags(tracer):
     if priority_sampling:
         opts["priority_sampling"] = asbool(priority_sampling)
 
-    opts["collect_metrics"] = asbool(get_env("runtime_metrics", "enabled"))
+    opts["collect_metrics"] = True
```

Comment:
I think it is more user friendly to have a `LIGHTSTEP_METRICS_ENABLE` if possible, so the user can deal with a `True` and not with a `False`. Also, if not possible, it would be cleaner to just do:

```python
opts["collect_metrics"] = not asbool(os.environ.get("LIGHTSTEP_METRICS_DISABLE"))
```

Reply:
We want to have it enabled by default, hence the environment variable name. Added a test and updated the code to match suggestion 👍